runHarness/WorkloadDrivers/AuctionWorkloadDriver.pm

```diff
@@ -1144,33 +1154,12 @@ sub startRun {
 					my $runLengthMinutes = $totalTime / 60;
 					my $impl             = $self->getParamValue('workloadImpl');
 
-					if (!$usingFindMaxLoadPathType) {
+					if ($usingFixedLoadPathType) {
 						$console_logger->info(
 							"Running Workload $workloadNum: $impl.  Run will finish in approximately $runLengthMinutes minutes."
```

Comment:
$runLengthMinutes should be rounded for this log message to prevent long strings of decimals when it is not divisible by 60.
Running Workload 1: auction.  Run will finish in approximately 6.33333333333333 minutes.

Reply:
Good catch.  Will fix.